[#1] PR Template과 Issue Template 생성

ksundong opened the pull request:
Fixes #1 

## 설명

PR Template과 Issue Template을 생성합니다.
오늘 팀 빌딩에 나왔던 내용으로 작성하였습니다.

## 작업 유형

- [x] 신규 기능 추가

## 체크리스트

- [x] 이 PR의 코드들이 이 프로젝트의 스타일 가이드를 준수했는가?
- [x] 내 코드에 대한 자기 검토가 되었는가?

ksundong committed:
Team-#1 docs: PR template 추가

ksundong committed:
Team-#1 docs: BE issue template 추가

ksundong committed:
Team-#1 docs: FE issue template 추가

ksundong committed:
Team-#1 docs: Team issue template 작성

Close #1 issue.

ksundong committed:
Team-#1 style: 빈 라인 제거

reesekimm commented on .github/ISSUE_TEMPLATE/-team--title.md:
```diff
@@ -0,0 +1,14 @@
+---
+name: "기타 공통 이슈 템플릿"
+about: 기타 공통 이슈 템플릿
+title: "[Team] Title"
+labels: "🧑‍🤝‍🧑Team"
```
Team 라벨 emoji와 텍스트 사이 space 1개 추가해주실 수 있나요? (라벨은 수정해둔 상태입니다.)
추가할때 미처 신경을 못썼네요..ㅎㅎ

ksundong replied:
```suggestion
name: "기타 공통 이슈 템플릿"
about: 기타 공통 이슈 템플릿
title: "[Team] Title"
labels: "🧑‍🤝‍🧑 Team"
```

reesekimm commented on .github/ISSUE_TEMPLATE/-team--title.md:
```diff
@@ -0,0 +1,14 @@
+name: "기타 공통 이슈 템플릿"
+about: 기타 공통 이슈 템플릿
+title: "[Team] Title"
+labels: "🧑‍🤝‍🧑Team"
```
Team 라벨 emoji와 텍스트 사이 space 1개 추가해주실 수 있나요? (라벨은 수정해둔 상태입니다.)
추가할때 미처 신경을 못썼네요..ㅎㅎ

ksundong replied:
반영했습니다!

reesekimm reviewed (commented):
감사합니다!😃

ksundong committed:
Team-#1 fix: Team 라벨에 space 추가

https://github.com/codesquad-member-2020/issue-tracker-01/pull/3/files/b259b3a7d5645608bc48e691c8d973d2f9c57ca3#r436442149
